source/wp-content/themes/wporg-developer/inc/formatting.php

```diff
@@ -40,6 +40,7 @@ public static function do_init() {
 		add_filter( 'devhub-format-description', array( __CLASS__, 'fix_param_hash_formatting' ), 9 );
 		add_filter( 'devhub-format-description', array( __CLASS__, 'fix_param_description_html_as_code' ) );
 		add_filter( 'devhub-format-description', array( __CLASS__, 'convert_lists_to_markup' ) );
+		add_filter( 'devhub-format-hash-param-description', array( __CLASS__, 'fix_param_description_quotes_to_code' ) );
```

Comment:
Should this also be added to `devhub-format-description`?

Reply:
Yes, I think it does. Or more specifically, I think it should probably be moved to there, and only there now.

Originally I was having trouble with the amount of HTML in `devhub-format-description` (as it runs over the combined outputs from `devhub-format-hash-param-description`) causing it to match too much.

Now that it's splitting by HTML tags, that'll probably now work with less fuss, but it'll likely also need to detect when it's within `<pre>` or `<code>` tags and not affect the internal string values of those functions..

Thanks for mentioning this!

Reply:
[3ea0d11](https://github.com/WordPress/wporg-developer/pull/97/commits/3ea0d118f90072478b471816d40c9fdf51f70714)